app/src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -901,14 +886,8 @@ class Simpletask : ThemedNoActionBarActivity() {
         input.setText(mainFilter.proposedName)
 
         alert.setPositiveButton("Ok") { _, _ ->
-            val text = input.text
-            val value: String
-            if (text == null) {
-                value = ""
-            } else {
-                value = text.toString()
-            }
-            if (value == "") {
+            val value = input.text?.toString()
+            if (value.isNullOrBlank()) {
```

Comment:
Brittle + clarity. This actually doesn't compile (oops, I probably should have done this before pushing. On the other hand, it means the change was successful, in that the type system now catches errors like this). I'm about to push the fix, but I already started this review and didn't want to lose it.